plugins/modules/ec2_transit_gateway_info.py

```diff
@@ -77,65 +79,53 @@
         options:
             description: A dictionary of the transit gateway options.
             returned: always
-            type: complex
+            type: dict
             contains:
                  amazon_side_asn:
                     description:
-                      - A private Autonomous System Number (ASN) for  the  Amazon
-                        side  of  a  BGP session. The range is 64512 to 65534 for
-                        16-bit ASNs and 4200000000 to 4294967294 for 32-bit ASNs.
+                      - A private Autonomous System Number (ASN) for the Amazon ide of a BGP session.
+                      - The range is 64512 to 65534 for 16-bit ASNs and 4200000000 to 4294967294 for 32-bit ASNs.
                     returned: always
                     type: int
                     sample: 64512
                  auto_accept_shared_attachments:
-                    description:
-                       - Indicates whether attachment requests  are  automatically accepted.
+                    description: Indicates whether attachment requests are automatically accepted.
                     returned: always
                     type: str
                     sample: "enable"
                  default_route_table_association:
-                    description:
-                      - Indicates  whether resource attachments are automatically
-                        associated with the default association route table.
+                    description: Indicates whether resource attachments are automatically associated with the default association route table.
                     returned: always
                     type: str
                     sample: "disable"
                  association_default_route_table_id:
-                    description:
-                      - The ID of the default association route table.
+                    description: The ID of the default association route table.
                     returned: when present
                     type: str
-                    sample: "rtb-11223344"
+                    sample: "tgw-rtb-0fd332c911223344"
```

Comment:
AFAIU, Route Table ids are always of the format rtb-xxxx

Reply:
`association_default_route_table_id` is `ID of the default association route table`

<img width="468" alt="image" src="https://github.com/user-attachments/assets/0b62c0f2-9d9d-4911-a8e8-f2b1bb95a8c1">